CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ A breaking change should be clearly marked in this log.
 
 
 ## Unreleased
+* Include `expiration` attribute on ledger data entries in `getContractData` and `getLedgerEntries` responses ([#153](https://github.com/stellar/js-soroban-client/pull/153)).
```

Comment:
Thanks for adding this! Can you put it in an `### Added` section?

Reply:
yes, moved - https://github.com/stellar/js-soroban-client/pull/161/commits/b1a5641ac7a94b72c37bc55cf97e50a867126e5f